src/common/com/intellij/plugins/haxe/util/HaxeResolveUtil.java

```diff
@@ -424,7 +424,7 @@ public static HaxeClassResolveResult getHaxeClassResolveResult(@Nullable PsiElem
     if (element instanceof HaxeComponentName) {
       return getHaxeClassResolveResult(element.getParent(), specialization);
     }
-    if (element instanceof AbstractHaxeTypeDefImpl) {
+    if (element instanceof HaxeTypedefDeclaration) {
```

Comment:
Yes, they are the same "typedef". And that works even if we will check AbstractHaxeTypeDefImpl. For me checking to interface, not implementation is more natural, and later would be easier to searching similar checks. But here we gonna cast to impl for `getClassTarget()`, then I agree - my change is useless. Should I revert this line?